### ISSUE TITLE
force push issues

### ISSUE THREAD

**Issue by doggy8088** [githubteacher/intro-to-github#253]
What if someone push forcely and replace the remote repository, what about the missing commits that exists in the issue content?  does anyone get notified?

**Comment by githubteacher** [comment]
@doggy8088 Yeah, that's why we recommend against force pushes :wink: The comments themselves won't be removed, but yes, the commits will change.

Re: notifications, the best option would probably be to write an app that listens for a [PushEvent](https://developer.github.com/v3/activity/events/types/#pushevent) webhook, then delivers a notification however you see fit (email, using the GitHub API to open an Issue, &c.). The PushEvent payload includes a "forced" value; please see [the example](https://developer.github.com/v3/activity/events/types/#webhook-payload-example-15).

Incidentally, In GitHub Enterprise, the instance admin can disable force pushes altogether. We don't currently offer that on Dotcom via the UI, though you can [contact support](https://github.com/contact) and request they disable force pushes for a given repo.
